### PR TITLE
feat: added user profile github link

### PR DESCRIPTION
**Pull request by wawrzonnn**

### CODE REVIEW

**Review by hansac91 — APPROVED** [review]

**Review by Sumick — CHANGES_REQUESTED** [review]
User, and Profile relate to each other. You dont need to query them separately. You can include them in each other, and get from DB as a single record.

File: src/data/frontend/user/types.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface UserModel {
+  id: string
+  profileId: string
+  githubDetails: {
+    username: string | null
```
Comment (Sumick) [inline src/data/frontend/user/types.ts:5]: **nitpick:** username cannot be null when GighubDetails is created

**Review by Sumick — APPROVED** [review]
Looks good!

**Review by darionauta — APPROVED** [review]
OK

**Pull request merged**